my_dagster_project/repository.py

```diff
@@ -1,10 +1,47 @@
-from dagster import load_assets_from_package_module, repository, define_asset_job
+import os
+
+from dagster import define_asset_job, load_assets_from_package_module, repository, with_resources
+from dagster_aws.s3 import s3_pickle_io_manager, s3_resource
 
 from my_dagster_project import assets
 
 
 @repository
 def my_dagster_project():
+
+    # # To take advantage of Dagster's incremental re-execution functionality (e.g. retry from failure),
```

Comment:
I think talking about re-execution is a little confusing here, because users _will_ be able to re-execute the asset if it fails. They just won't be able to define a downstream asset that depends on this one.

Reply:
Separately, why two levels of comment indentation here?

Reply:
> users will be able to re-execute the asset if it fails. They just won't be able to define a downstream asset that depends on this one.

that was why we say "*incremental* re-execution" here. added " When you define a downstream asset that depends on another asset, ..."

> Separately, why two levels of comment indentation here?

because it differentiate the comment from code and users can uncomment the entire block -- i wonder if using triple quoted comments instead would make sense.